src/storybook/story-test-runner/extend-stories.ts

```diff
@@ -12,9 +12,30 @@ type StoryFile = { default: HermioneMetaConfig } & Record<string, HermioneStoryC
 
 export function extendStoriesFromStoryFile(stories: StorybookStory[]): StorybookStoryExtended[] {
     const storiesMap = getStoriesMap(stories);
-    const storyFile = getStoryFile(stories[0].absolutePath);
+    const storyPath = stories[0].absolutePath;
+    const storyFile = getStoryFile(storyPath);
     const withStoryFileExtendedStories = stories as StorybookStoryExtended[];
 
+    if (!storyFile) {
+        if (storyPath.endsWith(".tsx")) {
+            const msg = [
+                "[hermione-storybook]",
+                "reading .tsx story files is not supported.",
+                `"hermione" section is ignored in "${storyPath}"`,
+            ].join(" ");
+
+            console.warn(msg);
```

Comment:
I would probably write it using `chalk.red`. Most people do not see the usual log.

Reply:
Those will be on the start, so users would see it, if they encounter an error.
Also they may not want to use the section. Extra red logs would just scare them.